flex/http.py

```diff
@@ -120,10 +126,27 @@ def _normalize_python3_urllib_request(request):
     )
 
 
+def _normalize_tornado_request(request):
```

Comment:
Sounds good :+1:

I'm not too familiar with tox; do you have an example of a config that changes the installed packages like that?